src/components/deck/default-deck.js

```diff
@@ -62,10 +62,23 @@ export default function DefaultDeck({
     [overviewMode, toggleMode]
   );
 
+  const onMobileSlide = e => {
+    if (navigator.maxTouchPoints < 1) return;
+    switch (e.dir) {
+      case 'Left':
+        deck.current.advanceSlide();
+        break;
+      case 'Right':
+        deck.current.regressSlide();
```

Comment:
This is great because I wouldn't expect us to need to animate in reverse when you want to go back a slide.